src/Rx/Weno/indexRx.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
this scripts should be moved to interface subdirectory.
Would put it in:
interface/weno/
(also would change r to lowercase in indexRx.php)

Reply:
done